Makefile

```diff
@@ -167,16 +167,16 @@ debug-shared: shared
 static: lib/libsass.a
 shared: $(LIBRARIES)
 
-lib/libsass.a: $(COBJECTS) $(OBJECTS)
+lib:
 	$(MKDIR) lib
```

Comment:
(not sure if its applicable)
`mkdir C:\libsass 2> NUL`, redirect stderr to NUL (something analogous to, but not exactly, devnull?).

So even if it throws, it will keep continue.

More at: http://stackoverflow.com/q/4165387.